crypto/x509v3/v3_utl.c

```diff
@@ -222,7 +222,8 @@ static char *bignum_to_string(const BIGNUM *bn)
     return ret;
 }
 
-char *i2s_ASN1_ENUMERATED(X509V3_EXT_METHOD *method, const ASN1_ENUMERATED *a)
+char *i2s_ASN1_ENUMERATED(const X509V3_EXT_METHOD *method,
```

Comment:
There seem to be quite a few functions that don't use all the parameters. Am I missing the reason for this?